errortypes/code.go

```diff
@@ -17,6 +17,8 @@ const (
 const (
 	UnknownWarningCode               = 10999
 	InvalidPrivacyConsentWarningCode = iota + 10000
+	AccountLevelDebugDisabledWarningCode
+	BidderLevelDebugDisabledWarningCode
```

Comment:
These values are 10002 and 10003. According to https://github.com/prebid/prebid-server/issues/1650 these values should be 10001 and 10002. Is it ok?

Reply:
Does that mean `InvalidPrivacyConsentWarningCode` is set to 10001?

Reply:
Yes, `InvalidPrivacyConsentWarningCode` is now 10001. 
If I comment out `//UnknownWarningCode               = 10999` then `InvalidPrivacyConsentWarningCode` is  10000. Looks like it depends on a position of all declared constants.

Reply:
I suspect that it is line 18 that is occupying the iota=0 spot, so at line 19 iota=1 ... would fix it, and probably cleaner, to move the `UnknownWarningCode` to the bottom of the list ... would be where it belongs sequentially anyway.

Reply:
Done

Reply:
Reverted back per morning discussion with team.